mono/mini/driver.c

```diff
@@ -2006,6 +2020,8 @@ mono_main (int argc, char* argv[])
 	/* Set rootdir before loading config */
 	mono_set_rootdir ();
 
+	mini_threads_runtime_init ();
+
 	if (enable_profile) {
```

Comment:
Still don't understand why profiling loading can't be a normal part of runtime initialization.